torch/csrc/distributed/c10d/reducer.cpp

```diff
@@ -582,9 +583,12 @@ void Reducer::mark_variable_ready(VariableIndex index) {
     divFactor_ = process_group_->getSize();
     auto& workHandle = forwardPassWorkHandle_.workHandle;
     if (workHandle) {
-      workHandle->wait();
-      at::Tensor& res = forwardPassWorkHandle_.resultTensor;
-      divFactor_ = res.item().to<int>();
+      auto useStaticWorldSize = forwardPassWorkHandle_.useStaticWorldSize;
```

Comment:
nit: Do we need the temporary `useStaticWorldSize` here?

Reply:
We can just use `forwardPassWorkHandle_.useStaticWorldSize`, but I thought a temporary would be a bit clearer.